l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -657,6 +688,21 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
         });
     }
 
+    /// @dev Receives and parses (name, symbol, decimals) from the token contract
+    function _getERC20Getters(address _token) internal view returns (bytes memory) {
```

Comment:
Can you move this to NTV, so that we only have a single instance of it?

Reply:
done